### DISCUSSION TITLE
115网盘上传不了文件

### DISCUSSION THREAD

**Discussion by qinguoluo** [AlistGo/alist#4783] [Q&A] [answered]
最近发现115网盘不能上传文件了，上传文件会报错：请升级到最新版本: unexpected error。但是alist明明是最新版。这个问题已经存在比较久了，最近alist更新的几个版本都没有解决这个问题。

另外从其他网盘复制文件到115也不行，报同样的错。有大佬知道这是怎么回事吗？

**Comment by qinguoluo** [comment]
是115的限制还是alist的问题，没有人知道吗？

**Comment by luxikakashi** [comment, marked as answer]
3.16.3版本解君愁

**Reply by qinguoluo** [reply]
已经回退到3.16.3，115可以上传了，顺便还解决了阿里云不能登陆的问题。我记得以前就出现过115不能上传的问题，后来版本是修复了的，再后来版本又重复出现这个问题，希望作者能够在将来的新版中修复。

**Comment by qinguoluo** [comment]
3.23.0出来了，115上传的问题还是没有修复，无奈又退回3.16.3。难道作者不打算修复这个问题了吗？

**Reply by luxikakashi** [reply]
新版已经修复啦
